src/app/components/flyout/index.html

```diff
@@ -1,17 +1,12 @@
 <sky-demo-page
   pageTitle="Flyout"
   skyuxModule="SkyFlyoutModule"
-  packageName="@skyux/flyout"
 >
 
   <sky-demo-page-summary>
     <p>
-      The flyout service launches a container over a page's content to display large quantities of supplementary information related to a task. The <stache-code>SkyFlyoutService</stache-code> provider lauches flyouts and provides a common look-and-feel. The <stache-code>SkyFlyoutInstance</stache-code> class watches for and triggers flyout events.
+      The flyout service launches a container over a page's content to display large quantities of supplementary information related to a task. The <stache-code>SkyFlyoutService</stache-code> provider lauches flyouts and provides a common look and feel. This service dynamically generates the flyout component and appends it directly to the document's <stache-code>body</stache-code> element. The <stache-code>SkyFlyoutInstance</stache-code> class watches for and triggers flyout events.
```

Comment:
I noticed a typo here - `lauches`. I think it was leftover from before?
Also, we are repetitive with that word in the first two sentences. Is there another way we could say this?

Reply:
Good catches. Thanks. ... For the repetition, I'll tighten things up and combine the second and third sentences.